projects/kinde-angular/src/lib/kinde-angular.service.ts

```diff
@@ -121,6 +133,12 @@ export class KindeAngularService implements OnDestroy {
     this.document.location.href = registerUrl.href;
   }
 
+  async register_with_org(options?: RegisterURLOptions): Promise<void> {
```

Comment:
Consistency thingy, everything is camelCase
```suggestion
  async registerWithOrg(options?: RegisterURLOptions): Promise<void> {
```